aviary/docs/user_guide/outputs_and_how_to_read_them.ipynb

```diff
@@ -176,7 +176,7 @@
    "outputs": [],
    "source": [
     "# Testing Cell\n",
-    "# ********NOTE********\n",
+    "# ****NOTE****\n",
     "# run with a dummy set of aviary_inputs and check that _list files are only generated for correct verbosity?"
```

Comment:
@johnjasa

Reply:
Yeah, that's a good idea. You could run with one of the pre-canned example cases and set verbosity then make sure that those lists are made. You probably don't actually need to run a full case; can just do 0 max iter or similar.